echopype/utils/prov.py

```diff
@@ -0,0 +1,31 @@
+from datetime import datetime as dt
+from typing import Dict
+
+from _echopype_version import version as ECHOPYPE_VERSION
+from typing_extensions import Literal
+
+ProcessType = Literal["conversion", "processing"]
+
+
+def echopype_prov_attrs(process_type: ProcessType, source_files: str = None) -> Dict[str, str]:
```

Comment:
Looking at this again and seeing how `processing_function` is currently separately done in `calibrate.api._compute_cal`, I wonder if we could rename this to `echopype_ver_time_prov` so that it is clear what this function does, for when we come back to provenance in the future.